rfc6962-bis.xml

```diff
@@ -656,7 +655,7 @@ updated on the fly.
 (see <xref target="cert_sctlist_extension"/>). This mechanism allows the use of
 unmodified TLS servers, but, because the included SCTs cannot be changed without
 re-issuing the certificate, increases the risk that the certificate will be
-refused if any of the SCTs become invalid.
+considered non-compliant if some of the SCTs become invalid (by the client not recognizing the logs they were issued from anymore).
```

Comment:
I've had a go at re-wording this paragraph.  How's this?
"This mechanism allows the use of unmodified TLS servers, but the SCTs cannot be updated on the fly.  Since the logs that signed the SCTs won't necessarily be accepted by TLS clients for the full lifetime of the certificate, there is a risk that TLS clients will subsequently consider the certificate to be non-compliant with this document and in need of re-issuance."